app/assets/javascripts/components/miller-columns.js

```diff
@@ -1 +1,88 @@
 //= require miller-columns-element
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function MillerColumns (module) {
+    this.module = module
+    this.searchable = module.getAttribute('data-searchable') === 'true'
+  }
+
+  MillerColumns.prototype.init = function () {
+    if (this.searchable) this.initSearch()
+  }
+
+  MillerColumns.prototype.initSearch = function () {
+    var element = this.module.querySelector('#js-app-c-miller-columns__search')
+    var input = this.module.querySelector('#js-app-c-miller-columns__search-input')
+    var millerColumns = this.module.querySelector('miller-columns')
+
+    if (!window.accessibleAutocomplete) {
+      element.parentNode.removeChild(element)
+      return
+    }
```

Comment:
Is this here to avoid JS errors if we've forgotten to include Autocomplete's JS?

If we think this is likely to be an issue, it might be nice to include a helpful error message. Otherwise it might end up being more confusing as it'll silently swallow the problem and look like it's doing 'nothing'.

```suggestion
    if (!window.accessibleAutocomplete) {
      element.parentNode.removeChild(element)
      console.error('Accessible Autocomplete is required if you want this Miller Columns instance to be searchable')
      return
    }
```

I'm also not sure what the benefit is of deleting the `#js-app-c-miller-columns__search` element here? It doesn't look like that `element` gets created by JS, so why do we need to remove it?

Reply:
I don't think we need the error message here as we have autocomplete component as part of the Whitehall application.

The reason why we need to remove the element is that there is an input field which needs to be removed. Also, this won't cause a JS error as it won't run if not available.